compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -4368,6 +4369,55 @@ public void visit(BLangStringTemplateLiteral stringTemplateLiteral) {
         result = rewriteExpr(constructStringTemplateConcatExpression(stringTemplateLiteral.exprs));
     }
 
+    @Override
+    public void visit(BLangRawTemplateLiteral rawTemplateLiteral) {
```

Comment:
Shall we add a comment explaining the final desugar outcome?

Reply:
Added with c860ced